includes/definitions/discovery/moxa-etherdevice.yaml

```diff
@@ -14,3 +18,61 @@ modules:
                     descr: 'Power Consumption'
                     index: 'powerConsumption.{{ $index }}'
                     value: powerConsumption
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::poePortConsumption
+                    num_oid: .1.3.6.1.4.1.8691.7.86.1.40.6.1.2.
+                    descr: '{{ $ifDescr }} PoE Consumption'
+                    index: 'poePortConsumption.{{ $index }}'
+                    value: poePortConsumption
+        temperature:
+            data:
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::sfpTemperature
+                    num_oid: .1.3.6.1.4.1.8691.7.86.1.10.7.1.3.
+                    descr: '{{ $ifDescr }} SFP module temperature'
+                    index: 'sfpTemperature.{{ $index }}'
+                    value: sfpTemperature
+        voltage:
+            data:
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::sfpVoltage
+                    num_oid: .1.3.6.1.4.1.8691.7.86.1.10.7.1.4.
+                    descr: '{{ $ifDescr }} SFP module Supply Voltage'
+                    index: 'sfpVoltage.{{ $index }}'
+                    value: sfpVoltage
+        dbm:
+            data:
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::sfpTxPower
+                    num_oid: .1.3.6.1.4.1.8691.7.86.1.10.7.1.5.
+                    descr: '{{ $ifDescr }} SFP module Transmit Power'
+                    index: 'sfpTxPower.{{ $index }}'
+                    value: sfpTxPower
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::sfpRXPower
+                    num_oid: .1.3.6.1.4.1.8691.7.86.1.10.7.1.6.
+                    descr: '{{ $ifDescr }} SFP module Receive Power'
+                    index: 'sfpRxPower.{{ $index }}'
+                    value: sfpRXPower
+        state:
+            data:
+                -
+                    oid: MOXA-EDSP510A8POE-MIB::power1InputStatus
+                    num_oid: .1.3.6.1.4.1.8691.7.86.1.10.1.
+                    descr: 'Power Input 1 Status'
+                    index: 'powerInputStatus.1'
```

Comment:
May as well change this to `index: 'powerInputStatus.{{ $index }}`'

Reply:
Input 1 and Input 2 have different OID-s and both have index 0 so it probably won't work.